src/librustc_hir/lang_items.rs

```diff
@@ -217,7 +217,8 @@ language_item_table! {
     PartialOrdTraitLangItem,     "partial_ord",        partial_ord_trait,       Target::Trait;
 
     ManageableContentsTraitLangItem, "manageable_contents", manageable_contents_trait, Target::Trait;
-    NewVecInCollectorLangItem,    "new_in_collector",       new_in_collector_fn,            Target::Fn;
+    VecWithCapacityGCFnLangItem,    "vec_with_capacity_gc",       vec_with_capacity_fn,            Target::Fn;
```

Comment:
Do these line up properly in the actual sources?

Reply:
Unfortunately I can't get them to line up with the other lang items unless I give them shorter names.

Reply:
Ah OK. Well let's not obsess about it. Let's move along :)